hr_holidays_public/models/hr_holidays_public.py

```diff
@@ -127,8 +127,8 @@ class HrHolidaysPublicLine(models.Model):
     _description = "Public Holidays Lines"
     _order = "date, name desc"
 
-    name = fields.Char("Name", required=True)
-    date = fields.Date("Date", required=True)
+    name = fields.Char(string="Name Line", required=True)
```

Comment:
Why changing this?

Reply:
> Why changing this?

to prevent next pylint-odoo error

![image](https://user-images.githubusercontent.com/55989612/149090412-869637f8-5af0-4f20-b825-2963d9e62738.png)

if there is a better way of avoiding it let me know and I will implement it like that

Reply:
You just need to remove `string="Name"`, not to change it. That's because the string is deduced from the field name.

Reply:
> You just need to remove `string="Name"`, not to change it. That's because the string is deduced from the field name.

done!